### PR TITLE
Feature/aws ecs/additional security groups

### PR DESCRIPTION
**Pull request by krantzinator**
Closes #1740

This PR adds a fixup commit to the above original PR. See that PR for more context.

### CODE REVIEW

File: builtin/aws/ecs/platform.go [hunk 1]
```diff
@@ -1524,6 +1529,9 @@ type Config struct {
 	// Subnets to place the service into. Defaults to the subnets in the default VPC.
 	Subnets []string `hcl:"subnets,optional"`
 
+	// Security Group ID to define existing security groups for ecs.
```
Comment (catsby) [inline builtin/aws/ecs/platform.go:1532]:
```suggestion
	// Security Group IDs of existing security groups to use for ECS.
```

File: builtin/aws/ecs/platform.go [hunk 2]
```diff
@@ -1524,6 +1529,9 @@ type Config struct {
...
+	SecurityGroups []*string `hcl:"security_groups,optional"`
```
Comment (catsby) [inline builtin/aws/ecs/platform.go:1533]: At the risk of being pedantic, should this be `SecurityGroupIDs`?

File: builtin/aws/ecs/platform.go [hunk 3]
```diff
@@ -1626,6 +1634,14 @@ deploy {
 		docs.Default("public subnets in the default VPC"),
 	)
 
+	doc.SetField(
+		"security_groups",
+		"Security Group IDs to define existing security groups for ecs",
```
Comment (catsby) [inline builtin/aws/ecs/platform.go:1639]:
```suggestion
		"Security Group IDs of existing security groups to use for ECS",
```

**Review by catsby — COMMENTED** [review]
👍 but I wonder if it should be `security_group_ids` to be clear on what the expected values are, since SGs are one of the few AWS things that use IDs over ARNs

File: builtin/aws/ecs/platform.go (hunk 2 above)
Comment (krantzinator) [reply builtin/aws/ecs/platform.go:1533]: Agreed!

**Review by izaaklauer — APPROVED** [review]

**Pull request merged**